ticdc/manage-ticdc.md

```diff
@@ -218,52 +218,121 @@ cdc cli changefeed list --pd=http://10.0.10.25:2379
-                        "ignore-txn-start-ts": null
-                }
+  "info": {
+    "sink-uri": "mysql://127.0.0.1:3306/?max-txn-row=20\u0026worker-number=4",
+    "opts": {},
+    "create-time": "2020-08-27T10:33:41.687983832+08:00",
+    "start-ts": 419036036249681921,
+    "target-ts": 0,
+    "admin-job-type": 0,
+    "sort-engine": "memory",
+    "sort-dir": ".",
+    "config": {
+      "case-sensitive": true,
+      "enable-old-value": false,
+      "filter": {
+        "rules": [
+          "*.*"
+        ],
+        "ignore-txn-start-ts": null,
+        "ddl-allow-list": null
+      },
+      "mounter": {
+        "worker-num": 16
+      },
+      "sink": {
+        "dispatchers": null,
+        "protocol": "default"
+      },
+      "cyclic-replication": {
+        "enable": false,
+        "replica-id": 0,
+        "filter-replica-ids": null,
+        "id-buckets": 0,
+        "sync-ddl": false
+      },
+      "scheduler": {
+        "type": "table-number",
+        "polling-time": -1
+      }
+    },
+    "state": "normal",
+    "history": null,
+    "error": null
+  },
+  "status": {
+    "resolved-ts": 419036036249681921,
+    "checkpoint-ts": 419036036249681921,
+    "admin-job-type": 0
+  },
+  "count": 0,
+  "task-status": [
+    {
+      "capture-id": "97173367-75dc-490c-ae2d-4e990f90da0f",
+      "status": {
+        "tables": {
+          "47": {
+            "start-ts": 419036036249681921,
+            "mark-table-id": 0
+          }
         },
-        "status": {
-                "resolved-ts": 415241860902289409,
-                "checkpoint-ts": 415241860640145409,
-                "admin-job-type": 0
-        }
+        "operation": null,
+        "admin-job-type": 0
+      }
+    }
+  ]
 }
 ```
 
-In the above command:
+In the command and result above:
 
-- `resolved-ts`: The largest transaction `TS` in the current `changefeed`. Note that this `TS` has been successfully sent from TiKV to TiCDC.
-- `checkpoint-ts`: The largest transaction `TS` in the current `changefeed` that has been successfully written to the downstream.
-- `admin-job-type`: The status of a `changefeed`:
-    - `0`: The state is normal.
-    - `1`: The task is paused. When the task is paused, all replicated `processor`s exit. The configuration and the replication status of the task are retained, so you can resume the task from `checkpiont-ts`.
-    - `2`: The task is resumed. The replication task resumes from `checkpoint-ts`.
-    - `3`: The task is removed. When the task is removed, all replicated `processor`s are ended, and the configuration information of the replication task is cleared up. Only the replication status is retained for later queries.
+- `info` is the replication configuration of the queried `changefeed`.
+- `status` is the replication state of the queried `changefeed`.
+    - `resolved-ts`: The largest transaction `TS` in the current `changefeed`. Note that this `TS` has been successfully sent from TiKV to TiCDC.
```

Comment:
I didn't modify this sentence already in this document because it avoids grammatical ambiguity:
+ `TS` that has been successfully sent from TiKV to TiCDC?
+ the current `changefeed` that has been successfully sent from TiKV to TiCDC?

Reply:
OK. Then please update L329. They look very similar.

Reply:
Done in https://github.com/pingcap/docs/pull/3752/commits/c67fcf38c873b370d64c42a21810a0a20c7da94d